### PR TITLE
Fixing date bug

### PR DESCRIPTION
**Pull request by votemike**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-5704?page=com.atlassian.jira.plugin.system.issuetabpanels%3Aall-tabpanel

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Ds2013/Molecule/DateList/DayDateListItemPresenter.php [hunk 1]
```diff
@@ -28,4 +30,17 @@ public function isGmt(): bool
...
         return $this->options['user_timezone'] == 'GMT';
     }
+
+    public function isLink(): bool
+    {
+        $network = $this->service->getNetwork();
+        if (is_null($network)) {
+            return false;
+        }
+        $broadcastDay = new BroadcastDay($this->datetime, $network->getMedium());
```
Comment (BjornTwachtmann) [inline src/Ds2013/Molecule/DateList/DayDateListItemPresenter.php]: Is this run for every item in the date nav? If so, creating a new Broadcast Day for each item seems unreasonably expensive. Could BroadcastDay be passed in instead?

Comment (votemike) [inline src/Ds2013/Molecule/DateList/DayDateListItemPresenter.php]: Nope. Needs to be done 15 times.

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Pull request merged**